docs/apache-airflow/index.rst

```diff
@@ -84,8 +84,7 @@ unit of work and continuity.
     tutorial_taskflow_api
     howto/index
     ui
-    concepts
-    scheduler
```

Comment:
Can you add redirects to [redirects.txt](https://github.com/apache/airflow/blob/master/docs/apache-airflow/redirects.txt)? This will ensure that links that use the `latest` version will still work after a new version is released.

Reply:
Great point, added.